database/factories/CartFactory.php

```diff
@@ -2,13 +2,13 @@
 
 use Illuminate\Support\Collection;
 use IndieHD\Velkart\Contracts\CartRepositoryContract;
-use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidFactoryInterface;
 
 $cartRepository = resolve(CartRepositoryContract::class);
 
 $factory->define($cartRepository->modelClass(), function (Faker\Generator $faker) {
 
-    $identifier = Uuid::uuid4();
+    $identifier = resolve(UuidFactoryInterface::class)->uuid4();
```

Comment:
if you resolve here it will resolve on every iteration, i would recommend taking this out of the callback

Reply:
I take that back, i suppose it will regardless ?

Reply:
You're correct; that should be moved outside of the closure, regardless of the factory's inner-workings. There is no good reason to resolve it inside, and it probably hurts performance one way or another.

Reply:
I just pushed a fix.